src/functions-and-arrays.js

```diff
@@ -1,119 +1,281 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
-
-
+function maxOfTwoNumbers(a, b) {
+  if (a > b) {
+    return a;
+  }
+  if (a < b) {
+    return b;
+  } else {
+    return a;
+    return b;
```

Comment:
You don't need the second return statement, because the function will end with the first one.